<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyPixelInspectionFormClass</class>
 <widget class="GUIFormBase" name="PropertyPixelInspectionFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PropertyPixelInspectionForm</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetLibList">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="frameLibFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>381</width>
     <height>461</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QPushButton" name="pbClearNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear new</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyPixel.qrc">
      <normaloff>:/Qtres/ClearNew.png</normaloff>:/Qtres/ClearNew.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLibName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Library Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUpdate">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyPixel.qrc">
      <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDelete">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyPixel.qrc">
      <normaloff>:/Qtres/Delete.png</normaloff>:/Qtres/Delete.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSaveNew">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save New</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyPixel.qrc">
      <normaloff>:/Qtres/SaveNew.png</normaloff>:/Qtres/SaveNew.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>名称</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLibID">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QCheckBox" name="chbDetailSettingNarrow">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbSearchDotTargetNarrow">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditLibID_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>狭域</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbSearchDotMasterNarrow">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>検査有効チェック</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>閾値作成時の周囲包括ドット</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>検査時の探索ドット</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>検出サイズ（ドット）</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>相対閾値</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbNGSizeNarrow">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsbRelativeNGThresholdNarrow">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbAbsoluteNGThresholdNarrow">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>絶対閾値</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLineEdit" name="EditLibID_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>広域</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>検出サイズ（ドット）</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbSearchDotMasterBroad">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="chbDetailSettingBroad">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>相対閾値</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>閾値作成時の周囲包括ドット</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>検査時の探索ドット</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbNGSizeBroad">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbSearchDotTargetBroad">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>検査有効チェック</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbAbsoluteNGThresholdBroad">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>絶対閾値</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsbRelativeNGThresholdBroad">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="chbIsCalcPixelInterPoration">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>ピクセル補間機能を有効にする</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PropertyPixel.qrc"/>
 </resources>
 <connections/>
</ui>
